LoopBack/LoopBack-Prefix.pch

```diff
@@ -3,6 +3,5 @@
 //
 
 #ifdef __OBJC__
-    #import <MobileCoreServices/MobileCoreServices.h>
```

Comment:
Would you please revert this change?
This prevents AFNetworking from performing "Automatic MIME type detection when uploading files".
https://github.com/strongloop/loopback-sdk-ios/blob/master/SLAFNetworking/SLAFHTTPClient.h#L89
You'll see the message as a warning when compiling the workspace.